packages/menu-bar/src/vaadin-menu-bar-buttons-mixin.js

```diff
@@ -40,6 +50,18 @@ export const ButtonsMixin = (superClass) =>
       super.connectedCallback();
 
       this._initButtonAttrs(this._overflow);
+
+      // Observe parent node resize to detect overflow
+      this.__parent = this.parentNode instanceof ShadowRoot ? this.parentNode.host : this.parentNode;
```

Comment:
What about the case where the menu-bar is moved into a different element? Should that update the subscription?

Reply:
I guess this is already covered, since the `disconnectedCallback` is called and then  the `connectedCallback` on the new parent, so the observer will be applied to it.

Reply:
Ah, I missed this fact. 👍